client/my-sites/backup/hooks.js

```diff
@@ -154,17 +154,16 @@ export const useFirstMatchingBackupAttempt = (
 // Tolerates null settings values, unlike the implementation in `calypso/components/site-offset`;
 // I don't want to disturb existing behavior, but we may want to come back later
 // and DRY up this bit of code.
-export const useDateWithOffset = ( date, shouldExecute = true ) => {
+export const useDateWithOffset = ( date, { shouldExecute = true, keepLocalTime = false } = {} ) => {
```

Comment:
`keepLocalTime` is going to be `false` by default to preserve its default behavior.